pt/index.md

```diff
@@ -1,24 +1,31 @@
 ---
 title: Elixir School
 layout: home
-version: 1.0.0
+version: 2.0.0
 ---
 
-[![Licença](//img.shields.io/badge/license-MIT-brightgreen.svg)](http://opensource.org/licenses/MIT)
-
-As lições sobre a linguagem de programação Elixir foram inspiradas na [Scala School](http://twitter.github.io/scala_school/) do Twitter.
-
-_Nós encorajamos a sua participação e colaboração!_
-
-## Sobre o Elixir
-
-"Elixir é uma linguagem dinâmica e funcional projetada para construir aplicações escaláveis e de fácil manutenção." — [elixir-lang.org](http://elixir-lang.org/)
-
-Elixir aproveita a massivamente testada máquina virtual do Erlang para construir sistemas distribuídos e tolerantes a falhas com baixa latência.
-
-__Características__:
-
-+ Escalável
-+ Tolerante a falhas
-+ Programação funcional
-+ Extensível
+Elixir School é o primeiro destino para pessoas procurando aprender e dominar a linguagem de programação Elixir.
+
+Se você é um veterano experiente ou esta é sua primeira vez, você encontrará o que precisa em lições e recursos auxiliares.
+
+Através do trabalho de voluntários o Elixir School foi traduzido para muitas línguas. Algumas das traduções incluem: [Việt ngữ][vi], [汉语][cn], [Español][es], [Slovenčina][sk], [日本語][ja], [Polski][pl], [Português][pt], [Русском][ru], [Bahasa Indonesia][id], [Bahasa Melayu][ms], [Українською][uk], [한국어][ko], [Italiano][it], [Deutsch][de], [বাংলা][bn], [Türkçe][tr], and [ภาษาไทย][th].
+
+Nós recebemos e encorajamos você a se juntar a nós em continuar a fazer o Elixir School ótimo por se envolver em [elixirschool/elixirschool](https://github.com/elixirschool/elixirschool)!
```

Comment:
I think this sentence would be better translated to something like this: "Nós lhe damos as boas-vindas e encorajamos você a se juntar a nós para continuar a melhorar o Elixir School em ..."